### PR TITLE
chore: Translations for the allergens taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the allergens taxonomy

### CODE REVIEW

File: taxonomies/allergens.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ stopwords:pt:traços,vestígios,vestígio,a,e,uma,em,ou,de,que,com,contém,cont
 
 en:none, allergens-free, without allergens, without, n, no, no allergens, 0
 fr:aucun, aucune, sans allergènes, sans, pas, pas d'allergènes, aucun allergène, 0
+it:no allergeni
```
Comment (stephanegigandet) [inline taxonomies/allergens.txt:13]: We need complete translations, in order to match what we can get in the Excel files from producers.

```suggestion
it:nessuno, nessuna, senza allergeni, no allergeni, no, n, 0
```

**Review by aleene — APPROVED** [review]

File: taxonomies/allergens.txt (hunk 1 above)
Comment (stephanegigandet) [reply taxonomies/allergens.txt:13]: @teolemon : why did you mark this resolved?

Comment (teolemon) [reply taxonomies/allergens.txt]: I tried to commit it actually

Comment (teolemon) [reply taxonomies/allergens.txt]: and it's done, so closing now

**Pull request merged**